### ISSUE TITLE
ImportError: cannot import name build_absolute_url

### ISSUE THREAD

**Issue by jazzzz** [Affirm/cabot-alert-mattermost#12]
Trying to install this plugin, I have this error:
```
  File "/usr/local/lib/python2.7/dist-packages/cabot_alert_mattermost/models.py", line 15, in <module>
    from cabot.cabotapp.utils import build_absolute_url
ImportError: cannot import name build_absolute_url
```
I could not find `build_absolute_url` in any version of Cabot. What do I need to do to use this plugin?

**Comment by Aloshi** [comment]
This is a helper from [our fork](https://github.com/Affirm/cabot/blob/master/cabot/cabotapp/utils.py#L7-L9):

```python
def build_absolute_url(relative_url):
    """Prepend https?://host to a url, useful for links going into emails"""
    return '{}://{}{}'.format(settings.WWW_SCHEME, settings.WWW_HTTP_HOST, relative_url)
```

**Comment by jazzzz** [comment]
Maybe you should mention that this plugin is only intended to work with your fork, or do you intend to make it work with upstream Cabot?

**Comment by Aloshi** [comment]
Yeah, we should mention that it'll only work with our fork - I'll update the README soon. Sorry for the confusion.

The main reason for the incompatibility is that some Mattermost configuration options are part of the `CheckGroupMixin` class, which are in `cabotapp` (much like the HipChat plugin). This was important because we needed each service to be able to alert to a different channel. As far as I know, the plugin system doesn't offer a way for plugins to integrate per-service configuration options.

If you (or anyone reading this) is interested in adapting this plugin to work with upstream, I think all you'd need to change is `build_absolute_url` and `_get_mm_api_for_service` (since `service.mattermost_instance` and `service.mattermost_channel_id` don't exist in upstream).
